build_defs/build_defs.bzl

```diff
@@ -42,14 +43,20 @@ def arcs_cc_schema(name, src, out = None):
 def arcs_kt_schema(name, src, out = None):
     """Generates a Kotlin file for the given .arcs schema file."""
     _run_schema2pkg(
-        name = name,
+        name = name + "_genrule",
         src = src,
         out = out,
         language_name = "Kotlin",
         language_flag = "--kotlin",
         file_extension = ".kt",
     )
 
+    kt_jvm_library(
+        name = name,
+        srcs = [out],
+        deps = ["@arcs//src/wasm/kotlin:arcs"]
```

Comment:
I see how this ties in to the particle / Wasm disentanglement work that you said I should do. I'll refer back here for reference.